.github/boring-cyborg.yml

```diff
@@ -169,20 +169,6 @@ firstPRWelcomeComment: >
 firstPRMergeComment: >
   Awesome work, congrats on your first merged pull request!
 
-insertIssueLinkInPrDescription:
-  descriptionIssuePlaceholderRegexp: "^Issue link: (.*)$"
-  matchers:
-    jiraIssueMatch:
-      titleIssueIdRegexp: \[(AIRFLOW-[0-9]{4})\]
-      descriptionIssueLink: "[${1}](https://issues.apache.org/jira/browse/${1})"
```

Comment:
@kaxil is it safe to leave this block in for our transition period where people still might be targeting Jiras?

Reply:
Yes, it is safe. If it doesn't find the Jira in the title it won't do anything. However we need to remove `verifyTitles` block